src/gsi/read_dbz_netcdf.f90

```diff
@@ -526,7 +526,8 @@ subroutine read_dbz_mrms_netcdf(nread,ndata,nodata,infile,obstype,lunout,sis,nob
                  
 		    if(thislon>=r360) thislon=thislon-r360
                     if(thislon<zero ) thislon=thislon+r360
-                 
+                    if(thislon>=r360 .or. thislat >90.0_r_kind) cycle
```

Comment:
Is it better to change the above "if thislon>=r360) thislon=thislon-r360 "  to "thislon = mod(thislon, r360);   ?  This along with the following " if(thislon<zero ) thislon=thislon+r360"  would make sure thislon is between 0  and 360.

Reply:
@TingLei-NOAA 
This is to handle the case when the longitude is missing (just cycle). This happened in my test case when using debug mode. The missing lon will cause the GSI horizontal interpolation failed. In release mode, this might not be an issue. I can keep the code unchanged as retro runs (GSI release mode) by others don't bother by this. 

This is also happened to FED obs. But my collaborator at GSL has modified the data process code. The updated code will not write an obs with missing values into netcdf file.

Reply:
OK, the current code is acceptable if it is to deal with possible missing value cases.